py-polars/polars/dependencies.py

```diff
@@ -232,6 +235,7 @@ def _check_for_pydantic(obj: Any) -> bool:
     "pydantic",
     "pyarrow",
     "zoneinfo",
+    "gevent",
```

Comment:
Might want to move this up to line 233 to keep list (section) sorted alphabetically.